src/cpp/session/modules/RStudioAPI.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * RStudioAPI.hpp
+ *
+ * Copyright (C) 2009-16 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+#ifndef SESSION_RSTUDIOAPI_HPP
+#define SESSION_RSTUDIOAPI_HPP
+
+namespace rstudio {
+namespace core {
+   class Error;
+}
+}
+
+namespace rstudio {
+namespace session {
+namespace modules {
+namespace rstudioapi {
+   
```

Comment:
Wasn't needed at the time of your review since `Api.R` is declared in `SessionMain`, but now we initialize other helper functions and has been brought back.